lib/progressive_hash_diff.rb

```diff
@@ -0,0 +1,29 @@
+class ProgressiveHashDiff
+  attr_reader :presenter, :previous_item, :current_item
```

Comment:
I've found that by moving this into private I'm unable to test the `previous_item=` and `current_item=` methods.  Is that ok?